.github/workflows/build-heretek-rocmfpx.yml

```
@@ -21,7 +21,7 @@ concurrency:
   # Include matrix dimension so apu and rdna builds run in parallel within
   # the same workflow run. Gate cancellation on PR events so a tag push
   # (`v*`) is never cancelled by an unrelated main push landing mid-build.
-  group: build-linux-${{ matrix.target_family }}-${{ github.ref }}
+  group: build-linux-${{ matrix.flavor }}-${{ github.ref }}
```

actionlint (1.7.12)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)